src/eko/io/struct.py

```diff
@@ -338,6 +342,52 @@ class AccessConfigs:
     """The path to the permanent object."""
     readonly: bool
     "Read-only flag"
+    open: bool
+    "EKO status"
+
+    def assert_open(self):
+        """Assert operator is open.
+
+        Raises
+        ------
+        exceptions.ClosedOperator
+            if operator is closed
+
+        """
+        if not self.open:
+            raise exceptions.ClosedOperator
+
+    def assert_writeable(self, msg: Optional[str] = None):
+        """Assert operator is writeable.
+
+        Raises
+        ------
+        exceptions.ClosedOperator
+            see :meth:`assert_open`
+        exceptions.ReadOnlyOperator
+            if operators has been declared read-only
+
+        """
+        if msg is None:
+            msg = ""
+
+        self.assert_open()
+        if self.readonly:
+            raise exceptions.ReadOnlyOperator(msg)
+
+    @property
+    def read(self):
```

Comment:
why do we need this alias?

Reply:
Because it pairs with `write`, that is a more complex condition then the bare attributes.